src/components/TopNavbar/TopNavbar.mdx

```diff
@@ -0,0 +1,41 @@
+import { Preview, Story, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs/blocks';
+import TopNavbar from './TopNavbar.vue';
+import MegaMenu from "../MegaMenu/MegaMenu.vue";
+import MegaMenuItem from "../MegaMenu/MegaMenuItem.vue";
+
+# Top Navbar
+
+This responsive component represents the fixed header of the webapp, with a default slot where to include (for example) the Lob logo, megaMenus, etc.
+
+<Preview>
+  <Story id="components-topnavbar--primary" />
+</Preview>
+
+## How to Use
+
+```html
+<topNavbar v-bind="$props">
```

Comment:
Let's remove the `v-bind="$props"` here since that's only for Storybook and not for how we'd actually use the component